packages/YoutubeVideo/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "@tds/core-youtube-video",
+  "version": "1.0.0-dev",
+  "description": "",
+  "main": "index.cjs.js",
+  "module": "index.es.js",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/telusdigital/tds-core.git"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "author": "TELUS digital",
+  "engines": {
+    "node": ">=8"
+  },
+  "bugs": {
+    "url": "https://github.com/telusdigital/tds-core/issues"
+  },
+  "homepage": "http://tds.telus.com",
+  "peerDependencies": {
+    "react": "^16.8.0",
```

Comment:
Can we use "^16.8.2" to be inline with our other styled component refactors?

Reply:
Yes, although I think we should be changing everything to 16.8.5 because of the newly added shallow rendering support on hooks right?

Reply:
Wouldn't we only need to use `^16.8.5` in https://github.com/telus/tds-core/blob/master/package.json?